[Docs]: update installation guide for AIBrix with Helm prerequisites …

haitwang-cloud opened the pull request:
## Pull Request Description
This PR enhances the AIBrix installation documentation by restructuring and improving the Helm installation guide. The changes provide clearer prerequisites and a more streamlined installation process to ensure users have a smooth setup experience.

## Related Issues
Resolves: https://github.com/vllm-project/aibrix/issues/1538

**Important: Before submitting, please complete the description above and review the checklist below.**

---

<details>
<summary><strong>Contribution Guidelines (Expand for Details)</strong></summary>

<p>We appreciate your contribution to aibrix! To ensure a smooth review process and maintain high code quality, please adhere to the following guidelines:</p>

<h3>Pull Request Title Format</h3>
<p>Your PR title should start with one of these prefixes to indicate the nature of the change:</p>
<ul>
    <li><code>[Bug]</code>: Corrections to existing functionality</li>
    <li><code>[CI]</code>: Changes to build process or CI pipeline</li>
    <li><code>[Docs]</code>: Updates or additions to documentation</li>
    <li><code>[API]</code>: Modifications to aibrix's API or interface</li>
    <li><code>[CLI]</code>: Changes or additions to the Command Line Interface</li>
    <li><code>[Misc]</code>: For changes not covered above (use sparingly)</li>
</ul>
<p><em>Note: For changes spanning multiple categories, use multiple prefixes in order of importance.</em></p>

<h3>Submission Checklist</h3>
<ul>
    <li>[ ] PR title includes appropriate prefix(es)</li>
    <li>[ ] Changes are clearly explained in the PR description</li>
    <li>[ ] New and existing tests pass successfully</li>
    <li>[ ] Code adheres to project style and best practices</li>
    <li>[ ] Documentation updated to reflect changes (if applicable)</li>
    <li>[ ] Thorough testing completed, no regressions introduced</li>
</ul>

<p>By submitting this PR, you confirm that you've read these guidelines and your changes align with the project's contribution standards.</p>

</details>

Jeffwan commented on docs/source/getting_started/installation/installation.rst:
```diff
@@ -8,17 +8,17 @@ This guide describes how to install AIBrix manifests in different platforms.
     If you already have these components installed, you can use corresponding manifest to skip them.
 
-
+~~~~~~~~~~~~
```
seems it's not aligned with the length of "stable version"

haitwang-cloud commented:
@Jeffwan  I have addressed your comment and verify the WIKI is functional working in [aibrix--1539.org.readthedocs.build](https://aibrix--1539.org.readthedocs.build/1539/getting_started/installation/installation.html), PTAL again.

Jeffwan commented:
<img width="320" height="140" alt="image" src="https://github.com/user-attachments/assets/397da851-9549-4581-bd01-d05c4df516c6" />

@haitwang-cloud I noticed you changed the length of the header lines. could you make them aligned, this is recommended rst pattern
